<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<!--adapted from QT NGL demo https://github.com/NCCA/QtNGL-->
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alex Christo Presents: Boids</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="s_mainWindowGridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="s_transformGB">
      <property name="title">
       <string>Parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>maximum speed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="_maxSpeed">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>maximum steering force</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="_maxSteering">
         <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>alignment strength</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="_alignment">
         <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>seperation strength</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="_seperation">
         <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>cohesion strength</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="_cohesion">
         <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>desired seperation</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSlider" name="_distance">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>45</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>neighborhood radius </string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSlider" name="_radius">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>90</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>flock size</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSlider" name="_size">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>3000</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>teleport on edge collision</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="_edges">
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>agent size</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QSlider" name="_agentSize">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>25</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>obsticles</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="_obsticles">
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
